2024-12-19 20:49:13,482 - utils - INFO - Configuration loaded from /share/home/lsy_chenyanchao/projects/model_test/config.yaml
2024-12-19 20:49:13,486 - utils - INFO - Created experiment directory: experiments/20241219_204913
2024-12-19 20:49:13,501 - data - INFO - Dataset loaded with 569 samples
/share/home/lsy_chenyanchao/software/miniconda3/envs/deeplearning/lib/python3.8/site-packages/torch/utils/data/dataloader.py:557: UserWarning: This DataLoader will create 4 worker processes in total. Our suggested max number of worker in current system is 1, which is smaller than what this DataLoader is going to create. Please be aware that excessive worker creation might get DataLoader running slow or even freeze, lower the worker number to avoid potential slowness/freeze if necessary.
  warnings.warn(_create_warning_msg(
2024-12-19 20:49:13,533 - data - INFO - Data split completed - Train: 398, Val: 85, Test: 86
2024-12-19 20:49:13,552 - model - INFO - MLP model built successfully
2024-12-19 20:49:16,808 - train - INFO - Trainer initialized. Using device: cuda
2024-12-19 20:49:16,808 - __main__ - ERROR - Error in main execution: 'num_epochs'
Traceback (most recent call last):
  File "/share/home/lsy_chenyanchao/projects/model_test/scr/main.py", line 177, in <module>
    main(args.config, args.mode)
  File "/share/home/lsy_chenyanchao/projects/model_test/scr/main.py", line 94, in main
    trainer.train()
  File "/share/home/lsy_chenyanchao/projects/model_test/scr/train.py", line 82, in train
    num_epochs = self.config['num_epochs']
KeyError: 'num_epochs'
Traceback (most recent call last):
  File "/share/home/lsy_chenyanchao/projects/model_test/scr/main.py", line 177, in <module>
    main(args.config, args.mode)
  File "/share/home/lsy_chenyanchao/projects/model_test/scr/main.py", line 94, in main
    trainer.train()
  File "/share/home/lsy_chenyanchao/projects/model_test/scr/train.py", line 82, in train
    num_epochs = self.config['num_epochs']
KeyError: 'num_epochs'
